src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -119,6 +141,64 @@ def _filter_experiences_by_region_or_country(
     return db.query(PrivacyExperience).filter(False)
 
 
+@router.get(
+    urls.PRIVACY_EXPERIENCE_META,
+    status_code=HTTP_200_OK,
+    response_model=Page[PrivacyExperienceMetaResponse],
+)
+@fides_limiter.limit(CONFIG.security.public_request_rate_limit)
+async def get_privacy_experience_meta(
+    *,
+    db: Session = Depends(deps.get_db),
+    params: Params = Depends(),
+    region: Optional[str] = None,
+    component: Optional[ComponentType] = None,
+    request: Request,  # required for rate limiting
+    response: Response,  # required for rate limiting
+) -> AbstractPage[PrivacyExperienceMetaResponse]:
```

Comment:
nit: i could be wrong, but isn't the return type annotation here technically `AbstractPage[PrivacyExperience]`? and then the `response_model` would be responsible for eliminating in the API response any additional fields on the returned `PrivacyExperience` that aren't part of the `PrivacyExperienceMetaResponse` model/schema?

it's a bit academic, and i may be wrong in understanding the functionality/intention! but i was scratching my head for a few seconds trying to reconcile the return type annotation and the objects actually being returned in the function.

Reply:
Thank you for catching, it should be PrivacyExperience, this was leftover from when I was still toying with it being just PrivacyExperienceMetaResponse and not wrapped within the PrivacyExperience, but then I thought this should truly be a subset of the PrivacyExperience endpoint contents.